### PR TITLE
terraform for rds

### PR DESCRIPTION
**Pull request by MichaelHassan**
Terraform to set up AWS RDS.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,4 +17,12 @@ __pycache__
 
 # Data
 *.csv
-*.json
\ No newline at end of file
+*.json
+
+# Terraform
+.terraform
```
Comment (Leon-JS02) [inline .gitignore:23]: Good idea adding these.

File: terraform/README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+Then run with `terraform apply`.
+
+After use, destroy with `terraform destroy`.
+
+## Architecture Diagram
+
+This image shows the architecture diagram for this project.
+
+<img src="../architecture_diagram.png" alt="Data Visualisation" width="600"/>
```
Comment (Leon-JS02) [inline terraform/README.md:34]: Why is the alt "Data Visualisation"?

**Review by Leon-JS02 — DISMISSED** [review]
A decent start on the terraform for the project.

**Review by paulauus — APPROVED** [review]
Improvements resolved - name of architecture diagram correct now.

**Review by Leon-JS02 — APPROVED** [review]
Re-approval, thanks for making that change.

**Pull request merged**